src/lib/slug.js

```diff
@@ -0,0 +1,10 @@
+import slug from 'slug';
```

Comment:
@vencax Just as a note here, one of the main reasons we removed the `slug` package before was that it was almost doubling the CMS bundle size because it included an entire Unicode lookup table. I think we would need to use a different package here, there were some options discussed in #607.

Reply:
Of course, if it was a plugin, you could use `slug` if you wanted, no problem there!